tools/fast/bin/localnet/main.go

```diff
@@ -394,37 +383,16 @@ func isEmpty(name string) (bool, error) {
 	return false, err // Either not empty or error, suits both cases
 }
 
-func getSectorSize(smallSectors bool) (uint64, error) {
-	rp := repo.NewInMemoryRepo()
-	blockstore := bstore.NewBlockstore(rp.Datastore())
-	blockservice := bserv.New(blockstore, offline.Exchange(blockstore))
-
+func getProofsMode(smallSectors bool) uint64 {
```

Comment:
Can we change this back to `getSectorSize`? It's more descriptive of the method.

Reply:
Or you can keep the name, and change the return type, and call `proofs.SectorSize` after the call to `getProofsMode`, you can then use the value to pass into `NewEnvironmentMemoryGenesis`.